### PR TITLE
feat(spanner): add samples for low-cost instances

### PR DESCRIPTION
**Pull request by hengfengli**

### CODE REVIEW

**Comment by hengfengli** [comment]
@tbpg I tried to run `goimports -w . && find . -name go.mod -execdir go mod tidy \;` locally but couldn't see any updates. Do you know why the `Lint` check fails?

**Comment by hengfengli** [comment]
After I upgraded the cloud spanner to 1.21.0, I run the command and I can see some updates (complained by the Lint check). However, the Lint check still fails. When I try to run the command again, I couldn't see any updates.

File: spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	"io"
+
+	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	instancepb "google.golang.org/genproto/googleapis/spanner/admin/instance/v1"
+)
+
+func createInstanceWithProcessingUnits(ctx context.Context, w io.Writer, projectID, instanceID string) error {
+	instanceAdmin, err := instance.NewInstanceAdminClient(ctx)
+	if err != nil {
+		return err
```
Comment (olavloite) [inline spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go:30]: nit: add context information to error

**Review by olavloite — APPROVED** [review]

**Comment by hengfengli** [comment]
@codyoss Can you please help us to merge this PR? Thanks!

File: spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"context"
+	"fmt"
+	"io"
+
+	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	instancepb "google.golang.org/genproto/googleapis/spanner/admin/instance/v1"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+func createInstanceWithProcessingUnits(ctx context.Context, w io.Writer, projectID, instanceID string) error {
```
Comment (codyoss) [inline spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go:28]: nit: Please declare [context as needed](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#declare-a-contextcontext-as-needed)

File: spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	instanceAdmin, err := instance.NewInstanceAdminClient(ctx)
```
Comment (codyoss) [inline spanner/spanner_snippets/spanner/spanner_create_instance_with_processing_units.go:29]: nit: Please provide [example args](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-snippets).

**Review by codyoss — CHANGES_REQUESTED** [review]
Just a couple of small things

**Review by codyoss — APPROVED** [review]
LGTM, thank you!

**Pull request merged**